unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs

```diff
@@ -252,9 +261,22 @@ void UpdateParcelHold()
             }
         }
 
+        private void OnKernelConfigChanged(KernelConfigModel current, KernelConfigModel previous)
+        {
+            Debug.Log("OnKernelConfigChanged");
+
+            validWorldRanges = current.validWorldRanges;;
+        }
+
         bool CoordinatesAreInsideTheWorld(int xCoord, int yCoord)
         {
-            return (Mathf.Abs(xCoord) <= WORLDMAP_WIDTH_IN_PARCELS / 2) && (Mathf.Abs(yCoord) <= WORLDMAP_WIDTH_IN_PARCELS / 2);
+            foreach(WorldRange worldRange in validWorldRanges)
+            {
+                if (worldRange.Contains(xCoord, yCoord)) {
```

Comment:
```suggestion
                if (worldRange.Contains(xCoord, yCoord)) {
```
This file needs formatting to bring the braces after the new line.